src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -63,8 +62,8 @@ public void beginProcessing(String charset) {
     }
 
     @Override
-    protected void processFiltered(File file, List<String> lines) {
-        detector.processLines(FileText.fromLines(file, lines));
+    protected void processFiltered(File file, FileText fileText) {
+        detector.processLines(fileText);
```

Comment:
please avoid usage of temporal object - List<String> lines , that is easy refactoring.